common/wireaddr.h

```diff
@@ -121,7 +121,10 @@ struct wireaddr_internal {
 		/* ADDR_INTERNAL_ALLPROTO */
 		u16 port;
 		/* ADDR_INTERNAL_AUTOTOR */
-		struct wireaddr torservice;
+		struct torservice {
+			struct wireaddr torservice;
```

Comment:
Might be useful to call this `address` instead since `->torservice.torservice` is weird imho

Reply:
Hoops  #3155 is with the doco is not merged yet.
mini review docu: --addr=**autotor:**[TORSERVICEIP][:TORSEVICEPORT]**:torport:**[EXTERNPORT]]

Reply:
I will rename struct `wireaddr torservice to struct wireaddr torservice_address`